.gitignore

```diff
@@ -14,7 +14,9 @@
 
 # Arduino libraries (if installed in the project directory)
 libraries/
-firmware/config/config.h
+firmware/config/config.h*
+firmware/config/orb_config.h*
```

Comment:
This also includes the template, which very much should not be under gitignore (you did not commit it here)